float32 X_w
float32 Y_w
float32 Z_w
float32 theta_dx_rad
